### PR TITLE
Enhancement: Updates sidebar to a new design

### PR DESCRIPTION
**Pull request by nithindavid**
Screenshot 2021-08-04 at 6 41 27 PM

### CODE REVIEW

File: app/javascript/dashboard/components/layout/AvailabilityStatus.vue [hunk 1]
```diff
@@ -81,11 +53,17 @@ export default {
...
+        this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
+          availabilityIndex
+        ]
+      );
       return this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
         availabilityIndex
       ];
     },
     currentUserAvailabilityStatus() {
+      console.log(this.currentUser.availability_status);
```
Comment (iamsivin) [inline app/javascript/dashboard/components/layout/AvailabilityStatus.vue:66]: Can you remove this line `console.log(this.currentUser.availability_status);`

File: app/javascript/dashboard/components/layout/AvailabilityStatus.vue [hunk 2]
```diff
@@ -81,11 +53,17 @@ export default {
       const availabilityIndex = AVAILABILITY_STATUS_KEYS.findIndex(
         key => key === this.currentUserAvailabilityStatus
       );
+      console.log(
+        this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
+          availabilityIndex
+        ]
+      );
```
Comment (nithindavid) [inline app/javascript/dashboard/components/layout/AvailabilityStatus.vue:60]:
```suggestion
```

File: app/javascript/dashboard/components/layout/AvailabilityStatus.vue (hunk 1 above)
Comment (nithindavid) [reply app/javascript/dashboard/components/layout/AvailabilityStatus.vue:61]:
```suggestion
```

File: app/javascript/dashboard/components/layout/SidebarItem.vue [hunk 3]
```diff
@@ -124,11 +113,24 @@ export default {
...
 <style lang="scss" scoped>
-@import '~dashboard/assets/scss/variables';
-
+.sidebar-item {
+  margin: var(--space-small) 0;
+}
 .sub-menu-title {
   display: flex;
   justify-content: space-between;
+
```
Comment (iamsivin) [inline app/javascript/dashboard/components/layout/SidebarItem.vue:122]: Remove this unused line
```suggestion

```

File: app/javascript/dashboard/components/layout/SidebarItem.vue [hunk 4]
```diff
@@ -124,11 +113,24 @@ export default {
...
+  padding: 0 var(--space-small);
+  margin-bottom: var(--space-smaller);
+
```
Comment (iamsivin) [inline app/javascript/dashboard/components/layout/SidebarItem.vue:125]: Remove this unused line
```suggestion

```

File: app/javascript/dashboard/modules/sidebar/components/Secondary.vue [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          v-if="shouldShowSidebarItem"
+          :key="inboxSection.toState"
+          :menu-item="inboxSection"
+        />
+        <sidebar-item
+          v-if="shouldShowSidebarItem"
+          :key="labelSection.toState"
+          :menu-item="labelSection"
+        />
+
```
Comment (iamsivin) [inline app/javascript/dashboard/modules/sidebar/components/Secondary.vue:25]:
```suggestion

```

**Review by muhsin-k — APPROVED** [review]
@nithindavid Looks good to me

**Pull request merged**